config/rewrites.d/api.rackspace.com.json

```diff
@@ -2,122 +2,120 @@
   "api.rackspace.com": [
       {
           "description": "Redirect root traffic to developer.rs.com/docs/",
-          "from": "^/$",
+          "from": "^\\/$",
           "to": "/docs/",
-          "toProtocol": "https",
-			    "toHostname": "developer.rackspace.com",
-			    "rewrite": false,
-			    "status": 302
+          "rewrite": false,
```

Comment:
This is the root redirect. Don't we need to keep the "toProtocol" and "toHostname" settings?